pkg/meta/boltdb/boltdb.go

```diff
@@ -507,7 +514,13 @@ func (bdw *BoltDB) SetRepoReference(repo string, reference string, manifestDiges
 		}
 
 		if _, ok := repoMeta.Statistics[manifestDigest.String()]; !ok {
-			repoMeta.Statistics[manifestDigest.String()] = mTypes.DescriptorStatistics{DownloadCount: 0}
+			repoMeta.Statistics[manifestDigest.String()] = mTypes.DescriptorStatistics{
+				DownloadCount:     0,
+				LastPullTimestamp: time.Time{},
+				PushTimestamp:     time.Now().Local(),
+				// Q do we care if it was pushed by sync or by metaDB.ParseStorage()?
```

Comment:
What happens right now if the image was added by sync? What would this variable value be?

Reply:
it will be "", anonymous